src/apps/fear-and-greed/index.tsx

```diff
@@ -0,0 +1,158 @@
+/**
+ * @name Fear and Greed Index
+ * @description Gm. This PillarX app serves as an example
+ * of how you can build a simple app within PillarX. It
+ * shows the use of fetching data from external sources with
+ * Axios, UI libraries like JoyUI and Google Charts, custom 
+ * fonts and animations, translations using React Spring and
+ * custom assets. Please feel free to use this as a template
+ * and visit https://docs.pillarx.app for more.
+ */
+
+// Core
+import React, { useEffect } from 'react';
+
+// UI
+import { Box, Grid, Typography } from '@mui/joy';
+import { animated, useSpring } from '@react-spring/web';
+import { Chart } from 'react-google-charts';
+import WebFont from 'webfontloader';
+
+// Utils
+import axios from 'axios';
+import { useTranslation } from 'react-i18next';
+
+// Local assets and styles
+import BackgroundImage from './assets/moroccan-flower-dark.webp';
+
+/**
+ * Define interface
+ * for API data
+ */
+interface ILatestData {
+  value: string;
+  value_classification: string;
+  timestamp: string;
+  time_until_update: string;
+}
+
+export const App = () => {
```

Comment:
no need to export as module as it won't be used, default export at the very bottom is the one used

Reply:
oops didn't notice that export